hack/setup-local-regional.sh

```diff
@@ -0,0 +1,111 @@
+}
+
+parseInstance() {
+    local name=$1
+    local instance_zone=$2
+    # Globals.
+    nodeTag=$(gcloud compute instances describe ${name} --zone ${instance_zone} --format='value(tags.items[0])')
+}
+
+clusterName="$1"
+clusterRegion="$2"
+
+if [ -z "${clusterName}" ]; then
+    echo "Usage: $0 CLUSTER_NAME [REGION]"
+    echo
+    echo "REGION should be a region for regional clusters (e.g., us-central1)."
+    echo "REGION is optional if there is only one cluster with CLUSTER_NAME"
+    exit 1
+fi
+
+# For regional clusters, location is a region
+fmt='value(networkConfig.network,networkConfig.subnetwork,location,selfLink,name)'
+if [ -z "$clusterRegion" ]; then
+    clusters=$(gcloud container clusters list --format="${fmt}" --filter="name:${clusterName}")
+else
+    clusters=$(gcloud container clusters list --format="${fmt}" --filter="name:${clusterName} location:${clusterRegion}")
+fi
+
+if [ $(echo "${clusters}" | grep -v '^$' | wc -l) -gt 1 ]; then
+    echo "ERROR: more than one cluster matches '${clusterName}'"
+    echo "Matching clusters:"
+    echo "${clusters}"
+    exit 1
+fi
+
+if [ -z "${clusters}" ]; then
+    echo "ERROR: No cluster '${clusterName}' found"
+    exit 1
+fi
+
+
+parseCluster ${clusters}
+
+# VM instances names created by gke are truncated up to 24 characters
+nodesPrefix=$(echo "gke-$clusterName" | head -c 24)
+
+# Get instance group URLs for the regional cluster
+instanceGroupUrls=$(gcloud container clusters describe $clusterName --region $clusterRegion --format='value(instanceGroupUrls)')
+
+if [ -z "${instanceGroupUrls}" ]; then
+  echo "ERROR: No instance groups found for cluster ${clusterName} in region ${clusterRegion}"
+  exit 1;
+fi
+
+# Iterate through instance group URLs to find a running instance
+instance=""
+instance_zone=""
+while read -r igUrl; do
+    igName=$(echo ${igUrl} | awk -F"/" '{print $NF}')
+    # Extract zone from the instance group URL
+    igZone=$(echo ${igUrl} | sed 's+.*/zones/\([^/]*\)/.*+\1+')
+    # List instances in the current instance group
+    running_instances=$(gcloud compute instance-groups list-instances ${igName} --zone ${igZone} --format='value(instance)' --filter='status:RUNNING')
+    if [ ! -z "${running_instances}" ]; then
+        # Pick the first running instance found
+        instance=$(echo "${running_instances}" | head -n 1)
+        instance_zone=${igZone}
+        break
+    fi
+done <<< "$(echo ${instanceGroupUrls} | tr ' ' '\n')"
+
+if [ -z "${instance}" ]; then
+  echo "ERROR: No running instances found in any instance group for cluster ${clusterName} in region ${region}"
+  exit 1;
+fi
+
+parseInstance ${instance} ${instance_zone}
+
+gceConf="/tmp/gce.conf"
+echo "Writing ${gceConf}"
+echo "----"
+cat <<EOF | tee ${gceConf}
+[global]
+token-url = nil
+project-id = ${project}
+network-name = ${net}
+subnetwork-name = ${subnet}
+node-tags = ${nodeTag}
+local-zone = ${instance_zone}
+EOF
+
+echo "----"
+echo "Run glbc with hack/run-glbc.sh"
```

Comment:
I know this is copied from `hack/setup-local.sh`, however `hack/run-glbc.sh` doesn't exist.
```suggestion
echo "Run glbc with hack/run-local-glbc.sh"
```

Reply:
Fixed